sma-webconnect.js

```diff
@@ -33,6 +33,32 @@ module.exports = function (RED) {
     }
   };
 
+  const sb = {
+    "id": "1",
+    "values": {
+      "6100_00464800": {
+        "name": "phase1_voltage",
+        "divider": 100
+      },
+      "6100_00465700": {
+        "name": "frequency",
+        "divider": 100
+      },
+      "6100_40465300": {
+        "name": "phase1_current",
+        "divider": 1000
+      },
+      "6400_0046C300": {
+        "name": "pv_gen_meter",
+        "divider": 1000
+      },
```

Comment:
I have also added "pv_gen_meter" as having total Wh produced will be more accurate then attempting to calculating Wh from multiple "power" measurements over time.

Reply:
Just a little naming suggestion here: What do you think of `generation_meter_total`?

And would you mind adding that one to the `sb_tripower` config as well. I'd like to keep those in sync as good as possible.

Reply:
I chose `pv_gen_meter` so it would be the same as the GUI but am happy to change it.
![image](https://user-images.githubusercontent.com/3389734/108127835-c1798180-70fb-11eb-9489-2b03c8c904cf.png)
Can you please confirm that `6400_0046C300` works for the Sunny Tripower as well?

Reply:
> I chose `pv_gen_meter` so it would be the same as the GUI but am happy to change it.

It may be a little nitpicky, but as all the all values are current values, I'd like to point at that this one is a total value.

> Can you please confirm that 6400_0046C300 works for the Sunny Tripower as well?

If tested that already, so I can confirm that it works for the Tripower. :+1: The same applies for the frequency (`6100_00465700`), please add this to the `sb_tripower` then as well. Thanks!